src/clickup/service.ts

```diff
@@ -0,0 +1,113 @@
+import { IMerjoonProjects, IMerjoonService, IMerjoonTasks, IMerjoonUsers } from '../common/types';
+import {
+  IClickUpMember,
+  IClickUpList,
+  IClickUpItem,
+  IClickUpTeam,
+} from './types';
+import { ClickUpTransformer } from './transformer';
+import { ClickUpApi } from './api';
+
+export class ClickUpService implements IMerjoonService {
+
+  protected teamIds: string[] | undefined;
```

Comment:
if it is smth or undefined then' let's use `?` to make it potionl instead:
`protected teamIds?: string[]`